static/app/views/replays/detail/trace/replayTransactionContext.tsx

```diff
@@ -82,6 +83,7 @@ const TxnContext = createContext<TxnContextProps>({
 function ReplayTransactionContext({children, replayRecord}: Options) {
   const api = useApi();
   const organization = useOrganization();
+  useEmitTimestampChanges();
```

Comment:
it couldn't stay in `replayContext` because it reads from the new `useCurrentTimeHook`, so it had to be a child of that. 

I only put it in this one spot because it's used to communicate between the trace tab and the replay, to make the vertical purple lines showup in the trace tab. 
There are a few places that create a `<ReplayContext>` provider, and this `replayTransactionContext` is a child of one of those callsites. I looked at the other callsites and I don't think this hook is needed there because those places are for the different, mostly static, replay rendering cases. For example there's the diff tools stuff, issue page snippet rendering, none of them show a trace-view and replay that need to be sync'd up.